File: SemTask25/Program.cs
int ReadData(string number)
{
    Console.WriteLine(number);
    return int.Parse(Console.ReadLine() ?? "0");
}
int exponentiation(int num1, int num2)
{
    int res =1;
    while (num2>0)
    {
        res*=num1;
        num2-=1;
    }
    return res;
}
int number1 = ReadData("Введите 1 число");
int number2 = ReadData("Введите 2 число");
 Console.WriteLine(exponentiation(number1,number2));

File: SemTask25/program.py
'''def exponentiation(numA, numB):
    res = 1
    while numB!=0:
       res*=numA
       numB-=1
    return res

number1, number2 = int(input('Введите число ')),int(input('Введите степень '))

print(exponentiation(number1,number2))'''

#25*

operation = input('Введите операцию ')

def operations(op):
    if operation == '+':
        return 'Вы выбрали сложение'
    elif operation == '-':
        return 'Вы выбрали вычитание'
    elif operation == '/':
        return 'Вы выбрали деление'
    elif operation == '*':
        return 'Вы выбрали умножение'
    elif operation == '**':
        return 'Вы выбрали возведение в степень'
    else:
        return 'Неизвестная операция'
print(operations(operation))


a = int(input('Введите первое число '))
b = int(input('Введение второе число '))
def Culc(a,b,operation):
    if operation == '+':
        return  a + b
    elif operation == '-':
        return  a - b
    elif operation == '/':
        return a / b
    elif operation == '*':
        return a * b
    elif operation == '**':
        return a**b
    else:
        return 'Неизвестная операция'

print('Результат будет' ,Culc(a,b,operation))